### ISSUE TITLE
Allow setting a default delimiter

### ISSUE THREAD

**Issue by flowrean** [aotimme/gocsv#19]
I work almost exclusively with semicolon-delimited CSVs and it's a bit frustrating to always have to use `gocsv delim` before and after the actual commands. In some cases, like joins, `gocsv delim` isn't really an option even. Would you consider supporting setting an alternative default delimiter, perhaps via an environment variable? Would you review a pull request that would add this feature?

Since I needed this feature quickly, I set the default delimiter to semicolon in a fork, but it would of course be nicer to make it configurable in the main repository. I think tabs are also very common as delimiter and that would be another use case.

**Comment by aotimme** [comment]
Sorry for not getting back to this in a long time. I think a good way to deliver this would be via an environment variable -- perhaps `GOCSV_DELIMITER`?

e.g.
```shell
export GOCSV_DELIMITER=";"
gocsv view semicolon-delimited-file.scsv
```

If that sounds like a good way to approach this, let me know and I can make that change.

**Comment by aotimme** [comment]
I'm open to throwing it away and taking another approach, but here's a pull request that would implement the `GOCSV_DELIMITER` environment variable: #33

**Comment by aotimme** [comment]
Closed by #33

**Comment by flowrean** [comment]
A very valuable addition. Appreciate it! 😄
